src/Model/TransformerInterface.php

```diff
@@ -41,7 +41,6 @@
  *     blocks: array<mixed>,
  *   }>,
  * }
- *
```

Comment:
I'm fixing this too, in my PR 😅

Reply:
With the new php-cs-fixer release I was wondering about updating our cs-standard @jordisala1991

Reply:
What do you want to change? It's fine by me, but until it gets updated, we should apply the fixes right?

Reply:
Please take a look at a proposal https://github.com/sonata-project/SonataAdminBundle/pull/7920

we can order the param/return/throw/phpstan-tags
and we can configure the separation to be similar to our current code style